learning/katas/go/Core Transforms/Map/ParDo Struct/task.md

```diff
@@ -18,16 +18,16 @@
 -->
 # ParDo - using a struct as a DoFn
 
-In previous katas, we provided our ParDo a `func` as a DoFn.  In this example, we will explore
-the use of a `struct` to in our transform.
+In previous katas, we provided `beam.ParDo` a `func` as a DoFn.  In this example, we will explore
```

Comment:
I think for this paragraph, it should be okay without the `beam.` prefix.
What do you think @lostluck?

Reply:
I guess I could go either way?

Including the identifier "beam." is more specific and I like it better visually. (especially with the `codefont` indication).

Reading it aloud it is certainly more of a mouthful than without "beam.", but that's should only be a problem for very novice readers who aren't subtising the words they read yet.